dev/benchmarks/complex_layout/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 50;
+	objectVersion = 54;
```

Comment:
Most of these files changes are the example/integration test apps being updated.